### PR TITLE
Remind about the upcoming interviews

### LINKED ISSUE #9: Remind about the upcoming interviews
There must be three reminders:
* 1 day in advance (let's say 6 p.m.);
* morning (let's say 7 a.m.) on the day of the interview;
* 1 hour in advance.

Use [APScheduler](https://apscheduler.readthedocs.io/en/v3.5.3/index.html) to schedule reminders.

### PR DESCRIPTION
**Pull request by ya7on**
close #9

### CODE REVIEW

File: server/huntflow_reloaded/scheduler.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        func,
+        'date',
+        run_date=date,
+        args=args,
+        kwargs=kwargs
+    )
+
+def create_event(ctx, message):
+    """
+    Create tasks in scheduler:
```
Comment (eugulixes) [inline server/huntflow_reloaded/scheduler.py]:
```
Schedules the events:
* remind an hour in advance;
* remind in the morning of the event day;
* remind in the evening of the event day.
```

File: server/huntflow_reloaded/scheduler.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import re
+import json
+from datetime import timedelta, datetime
+
+from apscheduler.schedulers.tornado import TornadoScheduler
+
+_SCHEDULER = TornadoScheduler()
+
+def add(date, func, args=(), kwargs={}):  # pylint: disable=dangerous-default-value
+    """ Wrapper for adding new task to scheduler """
```
Comment (eugulixes) [inline server/huntflow_reloaded/scheduler.py]: `Shortcut for adding new events.`

File: server/huntflow_reloaded/scheduler.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        second=0
+    )
+    add(evening_of_yesterday, notify_interview, args=args, kwargs=kwargs)
+
+def make():
+    """ Run scheduler workers """
+    _SCHEDULER.start()
+
+def notify_interview(message, ctx=None):
+    """ Function that will be called for notify about interview """
```
Comment (eugulixes) [inline server/huntflow_reloaded/scheduler.py]: `Invoked when the event comes.`

File: server/huntflow_reloaded/scheduler.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    evening_of_yesterday = date.replace(
+        day=date.day - 1,
+        hour=7,
+        minute=0,
+        second=0
+    )
+    add(evening_of_yesterday, notify_interview, args=args, kwargs=kwargs)
+
+def make():
+    """ Run scheduler workers """
```
Comment (eugulixes) [inline server/huntflow_reloaded/scheduler.py]: `Runs the scheduler workers.`

**Review by polina-popova — CHANGES_REQUESTED** [review]
While running I had a problem with the automatic determining of the local timezone.

**Review by polina-popova — CHANGES_REQUESTED** [review]
With current changes bot any time is sure that the interview is going to be in an hour (even if not)

**Review by polina-popova — APPROVED** [review]

**Pull request merged**